add-on/_locales/en/messages.json

```diff
@@ -275,6 +275,18 @@
     "message": "Redirect requests for IPFS resources to the Custom gateway",
     "description": "An option description on the Preferences screen (option_useCustomGateway_description)"
   },
+  "option_dnslinkRedirect_title": {
+    "message": "Force load from Custom Gateway",
```

Comment:
I agree with the language tweak here. This might be out of scope for this PR but do you think it's widely understood what a Custom Gateway is? I would be inclined to be more specific, such as: **Always load DNSLink websites via IPFS Companion's local gateway.** (if that is a correct statement?)

Reply:
Will tweak the label, but want to keep "force" for now to emphasize its "over the top" behavior (until we can redirect in a way that keeps origin sandbox) 

@cwaring ack, lets track that as part of https://github.com/ipfs-shipyard/ipfs-companion/issues/741